src/ShowForm.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+
+class ShowForm extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      content : props.item.content || '',
+    }
+  }
+
+  handleChange = (event) =>{
+    this.setState({
+      content : event.target.value
+    })
+  }
+
+  handleOnSave = () => {
+    const {handleSave} = this.props
+    handleSave({content : this.state.content})
+  }
+
+  render() {
+    const { handleCancel } = this.props
```

Comment:
func này sao em không cho ra ngoài phần render giống các func khác nhỉ

Reply:
func handleCancel đc truyền sang từ component cha ko nhận tham số vào giống như func handleSave sẽ nhận tham số nên em viết nó trong render cho ngắn hơn là viết 1 func nữa trong component con rồi lại gọi lại chính cái handleCancel đấy ms dùng đc func trong component con 
vi dụ: handleOnCancel = () => {
  const { handleCancel } = this.props
  handleCancel()
}

Reply:
giúp cho phần jsx dễ nhìn hơn ạ